### ISSUE TITLE
Missing Logic in AkkaPersistenceLivenessProbe 

### ISSUE THREAD

**Issue by Zetanova** [petabridge/akkadotnet-healthcheck#49]
The SuicideProbe class is to check the persistence journal state.
It recovers the last event and snapshot from the journal 
After it writes a new event and snapshot to the journal 
and deletes all old events and snapshots. 

The issue is that it already send a RecoveryStatus on successful revoery back 
without checking the success of the new persistet event or snapshot

And in the case when the journal success in only recovery (read-mode) and not in persist (write-mode)
then the RecoveryStatus will still be always successful.

The bottom line is that the write and the delete of new "hit" messages is somehow 
not used by the healthcheck itself and only makes a hit on one sector of the SSD every 10sec
